### PR TITLE
hotfix/SUIT-19732

### PR DESCRIPTION
**Pull request by Noorama**

### CODE REVIEW

File: lib/api/request.js [hunk 1]
```diff
@@ -18,19 +19,21 @@ const {captureException} = require('../utils/sentry/Raven');
...
-		if (!requestObject.headers) {
-			requestObject.headers = {};
-		}
-
 		if (!requestObject.headers['Content-Type']) {
 			requestObject.headers['Content-Type'] = 'application/json';
 		}
 	}
 
+	setUserAgent(requestObject.headers);
```
Comment (polomsky) [inline lib/api/request.js:35]: What is the purpose of this change? `setUserAgent` is already called inside `fetch` on line [40](https://github.com/SuitestAutomation/suitest-js-api/pull/505/files#diff-7ca33c3a208a43200fedd5490d451cc48602a0282ddade8bcacedc18f5d50ab7R40)

**Review by polomsky — CHANGES_REQUESTED** [review]

File: lib/api/request.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const SuitestError = require('../utils/SuitestError');
 const makeUrlFromArray = require('../utils/makeUrlFromArray');
 const {suitestServerError} = require('../texts');
 const {captureException} = require('../utils/sentry/Raven');
+const {setUserAgent} = require('../utils/fetch');
```
Comment (polomsky) [inline lib/api/request.js:12]: Already required on line 5. Deduplicate

**Review by polomsky — CHANGES_REQUESTED** [review]

File: lib/api/request.js (hunk 1 above)
Comment (Noorama) [reply lib/api/request.js:35]: I see. So these changes are not needed at all.
